### ISSUE TITLE
Highlight winning/not winning 1822 bids

### ISSUE THREAD

**Issue by thepackrat** [tobymao/18xx#4385]
It would be helpful to highlight in colour which items you are currently bidding on and winning/not winning in the 1822 auction rather than forcing the scrolling through the dozen or so items looking for which one you are in the bidlist for. Perhaps green/yellow colouring on the items?

**Comment by ChrisTheCat** [comment]
A suggestion - use colored fonts, with the same colors that are used for player colored tokens in the map view. All the orange bids are thepackrat, the blue bids are ChrisShaffer, etc.

**Comment by yzemaze** [comment]
Colored fonts would only work with a contrasting bg. Safe route is using colored bg which is simpler anyway. It looks ok on first blush, but ymmv of course, because it might be a bit too colorful.
tl;dr Would this work for you?

[with default colors]
player colors activated => colored bg according to profile config (suggestion by @waaayback and Chris)
![image](https://user-images.githubusercontent.com/33390595/110808801-9f030000-8284-11eb-836e-721d714b8d96.png).![image](https://user-images.githubusercontent.com/33390595/110808823-a32f1d80-8284-11eb-81a4-89735d41d146.png)

players colors deactivated => colored bg for active user only (packrat’s suggestion)
![image](https://user-images.githubusercontent.com/33390595/110808860-af1adf80-8284-11eb-91cb-dd74d9d8882a.png)
![image](https://user-images.githubusercontent.com/33390595/110808876-b215d000-8284-11eb-9eb0-c96a5ebe1c1c.png)

**Comment by magnusfalt** [comment]
I think its looking really good.

**Comment by thepackrat** [comment]
I think the just-current-played highlighting suits the flow I'd use and avoids too much colour. LOoks good with the background. 

22 might be a little about blocking others, but the very limited number of bid tokens quickly makes it really just about the places where you already are and whether you're winning or not.

**Comment by waaayback** [comment]
I think this looks good.  I prefer the "player colors activated" version, that's what I would see looking at the physical board.

(or can you handle both versions through a player's profile screen?)

**Comment by thepackrat** [comment]
It's harder to see winning/losing which is easier on the physical track.  If we did do the all players, could the names be arranged in a single vertical line, winner at the top?

**Comment by yzemaze** [comment]
> can you handle both versions through a player's profile screen?

map: player colors ✅ => colors configured in profile are used
map: player colors ❌ => winning == green, bidding == yellow

**Comment by waaayback** [comment]
packrat - the winning bid is the first one listed. Does that help?

**Comment by ChrisTheCat** [comment]
I agree with thepackrat, a stacked bid approach would be better than a line wrapped paragraph.

**Comment by yzemaze** [comment]
> could the names be arranged in a single vertical line, winner at the top?

You know the answer is "yes". It’s just a question of space efficiency and liking/hating tables, respectively. I think a table would work fine.

Thanks all for the input.
